projects/helper/chain/cosmos.js

```diff
@@ -232,16 +232,15 @@ async function queryManyContracts({ contracts = [], chain, data }) {
 
 async function queryContracts({ chain, codeId, }) {
   const res = []
-  const limit = 1000
-  let offset = 0
-  let paginationKey = undefined
+  const limit = 100
+  let paginationKey
 
   do {
-    let endpoint = `${getEndpoint(chain)}/cosmwasm/wasm/v1/code/${codeId}/contracts?pagination.limit=${limit}&pagination.offset=${offset}`
+    let endpoint = `${getEndpoint(chain)}/cosmwasm/wasm/v1/code/${codeId}/contracts?pagination.limit=${limit}${paginationKey ? `&pagination.key=${encodeURIComponent(paginationKey)}` : ''}`
+    console.log(endpoint);
```

Comment:
looks good, can you remove this line though

Reply:
ops, my bad

Reply:
thanks for the fix